redaxo/src/core/lib/util/socket/socket.php

```diff
@@ -114,9 +109,9 @@ public static function factoryUrl($url)
     /**
      * @return $this
      */
-    public function enableResponseContentEncoding(): self
+    public function acceptCompression(): self
     {
-        $this->responseContentEncoding = true;
+        $this->acceptCompression = true;
         $this->addHeader('Accept-Encoding', 'gzip, deflate');
```

Comment:
müssen wir sicherstellen dass die header nur einmalig hinzugefügt werden?

Reply:
Im Moment wäre es kein Problem, da die Header einfach in einem assoziativen Array gespeichert werden. Könnte nur problematisch werden, wenn sich die addHeader Methode ändert.

Reply:
ok dann lassen wir es so, danke.